lib/CLIManager.js

```diff
@@ -409,6 +437,9 @@ class CLIManager {
     let token;
 
     async.series([
+      (next) => {
+        this._logoutUserFromProfile(profileName, next);
```

Comment:
I would add a check to make sure the profile existed. Otherwise there would be no need to logout.

Reply:
the method we're calling here makes the check